codec-http2/src/main/java/io/netty/handler/codec/http2/HttpConversionUtil.java

```diff
@@ -251,6 +252,37 @@ public static FullHttpRequest toHttpRequest(int streamId, Http2Headers http2Head
     }
 
     /**
+     * Create a new object to contain the request data.
+     *
+     * @param streamId The stream associated with the request
+     * @param http2Headers The initial set of HTTP/2 headers to create the request with
+     * @param validateHttpHeaders <ul>
+     *        <li>{@code true} to validate HTTP headers in the http-codec</li>
+     *        <li>{@code false} not to validate HTTP headers in the http-codec</li>
+     *        </ul>
+     * @return A new request object which represents headers for a chunked request
+     * @throws Http2Exception see {@link #addHttp2ToHttpHeaders(int, Http2Headers, FullHttpMessage, boolean)}
+     */
+    public static HttpRequest toHttpChunkedRequest(int streamId, Http2Headers http2Headers, boolean validateHttpHeaders)
```

Comment:
consider renaming `toHttpRequest` -> `toFullHttpRequest` and renaming this `toHttpChunkedRequest` -> `toHttpRequest`. The "chunked" terminology may be confused with "chunked" encoding as defined in https://tools.ietf.org/html/rfc7230#section-4.1